src/inc/quic_platform_posix.h

```diff
@@ -1100,7 +1093,15 @@ CxPlatEventQInitialize(
     _Out_ CXPLAT_EVENTQ* queue
     )
 {
-    return (*queue = kqueue()) != -1;
+    // Create kqueue
+    if ((*queue = kqueue()) == -1) {
+        return false;
+    }
```

Comment:
I'm not sure about most of the changes in this file. Why exactly are they necessary? What happens without them?